pom.xml

```diff
@@ -63,7 +63,7 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <github.global.server>github</github.global.server>
     <site.installationModule>google-cloud-storage-parent</site.installationModule>
-    <google.core.version>1.92.5</google.core.version>
+    <google.core.version>1.92.6-SNAPSHOT</google.core.version>
```

Comment:
hard veto on this; we must not depend on snapshots. If this means we have to wait for a release of google core so be it.

Reply:
I'm doing this only for development*